ui/src/shared/components/inlineLabels/InlineLabels.tsx

```diff
@@ -32,19 +42,22 @@ export default class InlineLabels extends Component<Props> {
 
     return (
       <div className="inline-labels--container">
-        <InlineLabelsEditor
-          labels={labels}
-          selectedLabels={selectedLabels}
-          onAddLabel={onAddLabel}
-          onCreateLabel={onCreateLabel}
-        />
+        {this.isEditable && (
+          <InlineLabelsEditor
```

Comment:
If this is a long term thing, it could be nice to factor it out into a few different components, rather than one `InlineLabels` component that conditionally renders different stuff.

Maybe something like

- `EditableInlineLabels` which is composed of an`InlineLabels` and `InlineLabelsEditor`
- `InlineLabels`

Then the `TemplateCard` could render an `InlineLabels` directly.

Reply:
Went with the enum because I think of three modes we might need to be aware of:

- user can `create` labels and `read`, `add` and `remove` on a resource
- user can `read`, `add` and `remove` labels on a  resource
- user can `read` labels on a resource

I'll create an issue for `InlineLabels` in TechDebt because its component name should clarify it's handling these concerns.

Reply:
Note: I still have to verify that we'll want the user experience where only an org owner can add/remove labels. I think the intent was to have labels create/delete be restricted but generally allow users to add/remove labels. So I'm not sure atm.